docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -160,6 +160,18 @@ This is a known problem when using the `jsdom` library (a dependency of Jest) wi
 
 Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
 
+### `Module not found errors`
+When you use Connected components and you are supposed to see the folow errors from `aws-crt` and `encoding` on the server:
+
+```bash
+./node_modules/node-fetch/lib/index.js
+Module not found: Can't resolve 'encoding' in 'xxx'
+
+./node_modules/@aws-sdk/client-sts/node_modules/@aws-sdk/util-user-agent-node/dist-cjs/is-crt-available.js
+Module not found: Can't resolve 'aws-crt' in 'xxx'
+```
+
+It will not affect the use and functionality. This is a known transitive issue from SDKs. The issue is currently tracked [here](https://github.com/aws-amplify/amplify-js/issues/11030).
```

Comment:
What SDKs though? All SDKs or the AWS SDK?

Reply:
AWS SDKs

Reply:
```suggestion
It will not affect the use and functionality. This is a known transitive issue when using the AWS SDK. The issue is currently tracked [here](https://github.com/aws-amplify/amplify-js/issues/11030).
```